llpc/context/llpcComputeContext.h

```diff
@@ -59,6 +59,9 @@ class ComputeContext : public PipelineContext {
   // Gets the count of active shader stages
   virtual unsigned getActiveShaderStageCount() const { return 1; }
 
+  // Check whether the pipeline uses features relevant to subgroup size
+  virtual bool useSubgroupSize() const;
```

Comment:
nit: `usesSubgroupSize`? `use` makes this function sound like it makes it use it instead of checking if it already does it